cmd/cloud-network-config-controller/main.go

```diff
@@ -268,7 +268,8 @@ func init() {
 	controllerNamespace = os.Getenv(controllerNamespaceEnvVar)
 	controllerName = os.Getenv(controllerNameEnvVar)
 	if controllerNamespace == "" || controllerName == "" {
-		klog.Exit("Controller ENV variables are empty: %q: %s, %q: %s, cannot initialize controller", controllerNamespaceEnvVar, controllerNamespace, controllerNameEnvVar, controllerName)
+		klog.Exit("Controller ENV variables are empty:", controllerNamespaceEnvVar, controllerNamespace, "%q:",
```

Comment:
I think this is not correct, "%q:" will just get printed as is.
Can't we just do:
```
klog.Exitf("Controller ENV variables are empty: %q: %s, %q: %s, cannot initialize controller", controllerNamespaceEnvVar, controllerNamespace, controllerNameEnvVar, controllerName)
```
?

Reply:
i over thought the errors. Exitf is all that was needed. updated.